wordpress-mobile/WordPress-iOS
# Issue/8687 Async Announcement Alert

Implements #8687.

This PR adds a fancy alert announcement to Aztec, which is displayed the first time you tap the Async Publish button. Later, we'll move this to the normal Publish button when we implement #8978.

![simulator screen shot - iphone 8 - 2018-03-29 at 23 17 32](https://user-images.githubusercontent.com/4780/38116466-14daa6a4-33a8-11e8-8a5b-47a4328219f8.png)

**To test:**

* Repeat the following steps multiple times to check each feature of the alert. You'll need to either use a fresh install each time, or replace line 1262 in `AztecPostViewController.swift` to override the user defaults check and get the alert to re-appear:
* Open the editor and add a title / text.
* Use the Async Publish button.
* Check:
  * The alert only appears once. It doesn't reappear if you follow the same flow again.
  * The "Keep editing" button just dismisses the alert.
  * The "How does it work?" button should show a web view. We'll need to change this URL before launch.
  * The "Publish Now" button should dismiss the alert and the editor and do an async publish.
